src/com/akshay/captain/jira.main.kts

```diff
@@ -245,3 +473,49 @@ class BasicAuthInterceptor : Interceptor {
         return chain.proceed(authenticatedRequest)
     }
 }
+
+/**
+ * Describes the representation
+ * of a "Webhook Url"
+ */
+data class Url(
```

Comment:
nit: move this data class to the top of the file or move it to a separate one

Reply:
I can move to top but cannot export to a different file as a script file it needs to be in same one.